framework/db/ActiveRecord.php

```diff
@@ -207,7 +207,11 @@ protected static function findByCondition($condition)
     protected static function filterCondition(array $condition)
     {
         $result = [];
+        // valid column names are table column names or column names prefixed with table name
         $columnNames = static::getTableSchema()->getColumnNames();
+        $columnNames = array_merge($columnNames, array_map(function($columnName) {
+            return static::tableName() . ".$columnName";
```

Comment:
`static::tableName()` could be cached in variable outside anonymous function for performance reasons. Micro-optimization, but anyway

Reply:
fixed.